src/borg/testsuite/archiver.py

```diff
@@ -3765,6 +3765,23 @@ def test_change_location_does_not_change_algorithm_argon2(self):
             key = msgpack.unpackb(a2b_base64(repository.load_key()))
             assert key["algorithm"] == "argon2 chacha20-poly1305"
 
+    def test_file_status_counters(self):
+        """Test file status counters in the stats of `borg create --stats`"""
+        # Test case set up: create a repository
+        self.cmd(f"--repo={self.repository_location}", "rcreate", KF_ENCRYPTION)
+        # Create an archive
+        result = self.cmd(f"--repo={self.repository_location}", "create", "--stats", "test_archive", "./")
+        result = result.split("\n")
+        result = [l.split(":") for l in result]
+        result = [l for l in result if l[0] in ("Added files", "Unchanged files", "Modified files")]
+        # The line above left the value to the right of the `:` with a leading space.
+        # The line below removes the space and converts the value to int.
+        result = [[l[0], int(l[1][1:])] for l in result]
+        result = {l[0]:l[1] for l in result}
+        assert result["Added files"] == 3
+        assert result["Unchanged files"] == 0
+        assert result["Modified files"] == 0
+
```

Comment:
ok, this looks quite good if that test already works.

after this initial backup, you could just add 1 more files and modifiy one existing file and check if the values behave accordingly.